src/App/App.tsx

```diff
@@ -1,9 +1,40 @@
 import * as React from 'react';
-import {MainRoutes} from './MainRoutes'
+import { Navigation } from '../shell/navigation/index';
+import { IRoute } from '../shell/navigation/navigationItem/NavigationItem';
+import { Finance } from '../Modules/Finance';
+import { Main } from '../Modules/Main';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+interface IAppState {
+    routes: IRoute[]
+}
+
+class App extends React.Component<any, IAppState> {
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            routes: [
+                { title: 'finance', path: '/finance', component: Finance },
+                { title: 'home', path: '/', component: Main }
+            ]
+        }
+    }
 
```

Comment:
As a shorter variant we can use:
public state = {}

instead of 
constructor(props) { this.state ={} }

Reply:
Done.